wizard_builder/tests/test_app/settings.py

```diff
@@ -0,0 +1,30 @@
+import os
+import sys
+
+# add wizard_builder folder to path
+# needed to add wizard_builder to INSTALLED_APPS
+base_dir = os.path.join(os.path.abspath(os.path.dirname(__file__)), '../../../')
+sys.path.append(base_dir)
+
+DEBUG = True
+
+USE_TZ = True
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql_psycopg2",
```

Comment:
at some point (I'm unclear if this is still the case) I was getting issues because the subprocess and the parent process were trying to access the sqlite db simultaneously https://docs.djangoproject.com/en/dev/ref/databases/#database-is-locked-errors

I'll check if that's still the case, I've swapped to test since then to use subprocesses a bit less